ChatApp/ChatApp.API/Mapper/UserProfile.cs

```diff
@@ -0,0 +1,58 @@
+﻿using AutoMapper;
+using ChatApp.API.ViewModels.User;
+using ChatApp.Core.DTO;
+using ChatApp.Core.Entities;
+using ChatApp.CQRS.Commands.Users;
+using MongoDB.Bson;
+
+namespace ChatApp.API.Mapper
+{
+    public class UserProfile : Profile
+    {
+        public UserProfile()
+        {
+            CreateMap<User, UserViewModel>()
+                .ConvertUsing(src => new UserViewModel
+                {
+                    Id = src.Id.ToString(),
+                    FirstName = src.FirstName,
+                    LastName = src.LastName,
+                    EmailAddress = src.EmailAddress,
+                    TelephoneNumber = src.TelephoneNumber,
+                    UserStatus = src.UserStatus
+                });
+
+            CreateMap<UserDTO, User>()
+                .ConvertUsing(src => new User
+                {
+                    Id = src.Id.ToString(),
+                    FirstName = src.FirstName,
+                    LastName = src.LastName,
+                    TelephoneNumber = src.TelephoneNumber,
+                    EmailAddress = src.EmailAddress,
+                    UserStatus = src.UserStatus
+                    //ADD CHATS and CREATED CHATS
```

Comment:
Add such comments in this way:
TODO: Add chats and created chats
Also it is a good practise to add jira code of task 
TODO: [JIRA-Code] Add ...